src/common/lib/client/defaultrealtime.ts

```diff
@@ -22,14 +22,15 @@ import Defaults from '../util/defaults';
  `DefaultRealtime` is the class that the non tree-shakable version of the SDK exports as `Realtime`. It ensures that this version of the SDK includes all of the functionality which is optionally available in the tree-shakable version.
  */
 export class DefaultRealtime extends BaseRealtime {
-  constructor(options: ClientOptions | string) {
+  // The public typings declare that this requires an argument to be passed, but since we want to emit a good error message in the case where a non-TypeScript user does not pass an argument, tell the compiler that this is possible so that it forces us to handle it.
```

Comment:
(this is super minor so feel free to ignore) we can probably dedupe comments in `DefaultRealtime` and `DefaultRest` by declaring an interface which they both implement

Reply:
I started trying to implement this and it wasn't immediately obvious how to do it in a friction-free way so I gave up; I don't really want to spend more time on this so think we can live with the duplication.